### ISSUE TITLE
Aria-required property is missing for required field in Admin screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#871]
**Describe the bug**
Aria-required property is missing for required field.

**To Reproduce**

1. Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test10&path=%2FSites%2Ftest10%2F&pathType=page)
2. Click on Add New Workflow button. It will open a pop up.
3. "Name" is a required field. Inspect the "Name" text box element, you will find that tha Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
